website/docs/cloud-docs/workspaces/settings/run-tasks.mdx

```diff
@@ -62,7 +62,7 @@ To create a new run task:
 
 ## Understanding Run Tasks Within a Run
 
-Run tasks perform actions between the [plan](/cloud-docs/run/states#4-the-plan-stage) and [apply](/cloud-docs/run/states#8-the-apply-stage) stages of a [Terraform run](/cloud-docs/run). Once all run tasks have completed, the run will end based on the most restrictive enforcement level in each associated run task.
+Run tasks perform actions between the [plan](/cloud-docs/run/states#4-the-plan-stage) and [apply](/cloud-docs/run/states#9-the-apply-stage) stages of a [Terraform run](/cloud-docs/run). Once all run tasks have completed, the run will end based on the most restrictive enforcement level in each associated run task.
 
 For example, if a mandatory task fails and an advisory task succeeds, the run will fail. If an advisory task fails, but a mandatory task succeeds, the run will succeed and proceed to the apply stage. Regardless of the exit status of a task, Terraform Cloud displays the status and any related message data in the UI.
```

Comment:
```suggestion
For example, if a mandatory task fails and an advisory task succeeds, the run fails. If an advisory task fails, but a mandatory task succeeds, the run succeeds and proceeds to the apply stage. Regardless of the exit status of a task, Terraform Cloud displays the status and any related message data in the UI.
```

Suggest removing 2 instances of "will" and changing associated verbs to present tense to avoid future tense and to keep consistent with the present tense in the rest of the sentences.